### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by KerimKochekov**

### CODE REVIEW

File: server/models/solution.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ var SolutionSchema = new Schema({
     enum: ["submitted", "ok", "failed", "running"],
     default: "submitted",
   },
+  time: {
+    type: Boolean,
+    required: true,
+    default: -1,
```
Comment (tracy2811) [inline server/models/solution.js:17]: - Why time is of `Boolean` type when you have a default value of `-1`?
- If it is `required`, you have to update tests.

File: server/views/solution_list.pug [hunk 2]
```diff
@@ -2,11 +2,16 @@ extends layout
...
 block content
   h1= title
-  
   ul
-    each solution in solution_list
+    each solution in solution_list.reverse()
       li 
-        a(href=solution.url) #{solution._id}
+        a(href=solution.url) #{solution._id} 
+        img(src="/stylesheets/"+solution.status+".png")
```
Comment (tracy2811) [inline server/views/solution_list.pug:9]: Why using `.png` file for icon?

File: server/views/solution_list.pug [hunk 3]
```diff
@@ -2,11 +2,16 @@ extends layout
...
+    each solution in solution_list.reverse()
       li 
-        a(href=solution.url) #{solution._id}
+        a(href=solution.url) #{solution._id} 
+        img(src="/stylesheets/"+solution.status+".png") 
+        if solution.status == "ok"
+            b()    [#{solution.time} ms]
+        else if solution.status != "failed"
+            b() 
+            img(type="button" src="/stylesheets/donate.png")
```
Comment (tracy2811) [inline server/views/solution_list.pug:14]: `img` tag doesn't have `type` attribute, wrap it inside a `button` tag instead.

**Review by tracy2811 — CHANGES_REQUESTED** [review]

File: server/views/solution_list.pug (hunk 3 above)
Comment (KerimKochekov) [reply server/views/solution_list.pug:14]: it has actually

File: server/models/solution.js (hunk 1 above)
Comment (KerimKochekov) [reply server/models/solution.js:17]: I forgot that, it should be Number

File: server/views/solution_list.pug (hunk 3 above)
Comment (tracy2811) [reply server/views/solution_list.pug:14]: no, it doesn't, according to [w3schools](https://www.w3schools.com/tags/tag_img.asp)

File: server/views/solution_list.pug (hunk 2 above)
Comment (tracy2811) [reply server/views/solution_list.pug:9]: You can use svg icons, or simply use text with color as we did inside solution-detail.
If you wanna keep those images, for consistency, I suggest you also make changes on solution-detail page

**Pull request merged**